lib/commit-lint.ts

```diff
@@ -119,7 +119,8 @@ export class LintCommit {
                 this.lines[i].slice(11).match(/\s/)) {
                 this.block(`Lines in the body of the commit messages ${""
                     }should be wrapped between 60 and ${
-                    this.maxColumns} characters.`);
+                    this.maxColumns} characters.\n`
+                     + `Exceptions are: indented lines and very long URLs/words.`);
```

Comment:
This is misleading. We do not allow very long URLs or words unless they are in their own line.

Reply:
`"Single" words` maybe. I was just rephrasing the 'URL' example in the comment lines above.

Did you have a particular suggestion?

Reply:
How about "Indented lines, and lines without whitespace, are exempt from this check."?

I don't know. _You_ are the native speaker, not I.

Reply:
"and very long URLs/words" makes it sound as if the lines could still contain whitespace.

Reply:
I've updated, rebased and pushed the branch. I dropped the plurals (which could apply to the whole message being linted) and made the commit message be about individual 'lines'. I also included that the error message avoids mention of the preamble that can contain spaces.

Not quite sure which of the GitHub buttons needs pressing to get the PR to update for GGG.